src/main/kotlin/com/openlattice/data/storage/PostgresEntityDataQueryService.kt

```diff
@@ -91,7 +102,7 @@ class PostgresEntityDataQueryService(
         }
         return streamableEntitySet(
                 linkingIdsByEntitySetId, authorizedPropertyTypesByEntitySetId,
-                EnumSet.noneOf(MetadataOption::class.java), Optional.empty(), adapter, true
+                metadataOptions, Optional.empty(), adapter, true
```

Comment:
I think this needs a different adapter like implicitEntityValuesByIdWithLastWrite, when metadatoptions are not empty. And since linking entityset authorization changed, you can pass any of the authorized property maps from authorizedPropertyTypesByEntitySetId

Reply:
oooh good call

Reply:
I think it should be only used, when metadatoptions is not empty, because we call getLinkedEntityDataWithMetadata with empty metadataOptions too, but implicitEntityValuesByIdWithLastWrite expects the last_write column to be there

Reply:
+1 you right